core/deployment/src/main/java/io/quarkus/deployment/builditem/ConfigurationBuildItem.java

```diff
@@ -0,0 +1,19 @@
+package io.quarkus.deployment.builditem;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+import io.quarkus.deployment.configuration.BuildTimeConfigurationReader;
+
+/**
+ * The build item which carries the build time configuration.
+ */
+public final class ConfigurationBuildItem extends SimpleBuildItem {
```

Comment:
Why isn't this one called `BuildTimeConfigurationBuildItem`?

Reply:
Well it also carries the run time configuration definitions, so really it's a bit more general.  I guess the doc could use clarification.